pkg/build/graph/edges.go

```diff
@@ -56,7 +57,8 @@ func AddInputOutputEdges(g osgraph.MutableUniqueGraph, node *buildgraph.BuildCon
 			in := imagegraph.FindOrCreateSyntheticImageStreamTagNode(g, imagegraph.MakeImageStreamTagObjectMeta(defaultNamespace(from.Namespace, node.BuildConfig.Namespace), name, tag))
 			g.AddEdge(in, node, BuildInputImageEdgeKind)
 		case "ImageStreamImage":
-			glog.V(4).Infof("Ignoring ImageStreamImage reference in BuildConfig %s/%s", node.BuildConfig.Namespace, node.BuildConfig.Name)
+			in := imagegraph.EnsureImageStreamImageNode(g, defaultNamespace(from.Namespace, node.BuildConfig.Namespace), from.Name)
```

Comment:
the compelling reason is that this pattern didn't exist at the time I did these changes, it came in in a rebase apparently.  i'll look into it.